apiManager.ts

```diff
@@ -0,0 +1,37 @@
+class ApiManager {
+	private mainUrl: string;
+
+	constructor(mainUrl: string) {
+		this.mainUrl = mainUrl;
+	}
+
+	private fetchJson(mainUrl: string): Promise<any> {
+		return fetch(mainUrl)
```

Comment:
```suggestion
	private fetchJson(url: string): Promise<any> {
		return fetch(url)
```
Feels incorrect calling it the main one since in this scope it is only one and can be anything. Main or not.